### ISSUE TITLE
Element type is invalid: expected a string

### ISSUE THREAD

**Issue by pdavid0** [rusfearuth/react-native-auto-grow-textinput#5]
```
import {AutoGrowInputText} from "react-native-auto-grow-textinput";

<AutoGrowInputText
        placeholder="placeholder"
        maxHeight={120}
    />
```

```
ExceptionsManager.js:63 
Element type is invalid: expected a string (for built-in components) 
or a class/function (for composite components) but got: undefined.
Check the render method.
```
R : 0.41.2
RN : 15.5.4

**Comment by rusfearuth** [comment]
@pdavid0 Tnx for issue!

**Comment by joshuapinter** [comment]
Same thing here. 

Just importing it and using `<AutoGrowInputText placeholder='Some text here' />`.

![screenshot 2017-07-15 14 00 24](https://user-images.githubusercontent.com/180819/28242336-09a8110c-6966-11e7-84e9-f8fc3a177e88.png)

**Comment by rusfearuth** [comment]
@joshuapinter Could you show part of code?
How did you import library?

if you use `import AutoGrowTextInput from 'react-native-auto-grow-textinput';
`, it won't work.

**Comment by joshuapinter** [comment]
Gotcha! Because it's not the default, you have to put curly braces around it, like: 

```js
import {AutoGrowTextInput} from 'react-native-auto-grow-textinput';
```

I'll try that right now and report back.

**Comment by joshuapinter** [comment]
@rusfearuth Nope, same problem.

![screenshot 2017-07-18 21 50 14](https://user-images.githubusercontent.com/180819/28349967-17320c8e-6c03-11e7-8feb-55eed3b273c7.png)

**Comment by marlti7** [comment]
i have the same problem

**Comment by rusfearuth** [comment]
@joshuapinter could you share some code?

**Comment by rusfearuth** [comment]
Guys, Could you write RN versions? I've checked 1.1.2 on RN 0.43.x. It's work fine.

**Comment by joshuapinter** [comment]
@rusfearuth Not much code to share. It's about as basic as it gets.

```js
import { AutoGrowInputText } from 'react-native-auto-grow-textinput';

// In my render#return()
<AutoGrowInputText />
```

### Versions
`"react-native-auto-grow-textinput": "1.1.2"`
`"react-native": "0.42.3"`

**Comment by rusfearuth** [comment]
@joshuapinter Hm.. It's really strange. I can't repeat your case. Do you use `flow` for checking your code?

**Comment by joshuapinter** [comment]
@rusfearuth No, but it's isolated to just including `<AutoGrowInputText />`. I know how difficult it is dealing with issues that you can't reproduce so when I find more time I'll dig in deeper and try and find the root cause myself.

Let's leave this Issue open for now as a reminder, though.

**Comment by janaka120** [comment]
@rusfearuth, I got the same problem @joshuapinter had.
 react-native-auto-grow-textinput: 1.1.2 
 RN 0.38
OS: Android 7.0
Device: samsung A5

**Comment by chuson1996** [comment]
Some here, try both `import AutoGrowInputText from 'react-native-auto-grow-textinput';` and `import {AutoGrowInputText} from 'react-native-auto-grow-textinput';` but none works

**Comment by jeremistadler** [comment]
There is a bug in this package but the author did not accept my PR so I created a scoped package with a few fixes: `@jeremistadler/react-native-auto-grow-textinput`

**Comment by rusfearuth** [comment]
I didn't have enough time. Today I'll check our pr and give feedback

вт, 19 сент. 2017 г. в 17:31, Jeremi Stadler <notifications@github.com>:

**Comment by jeremistadler** [comment]
@rusfearuth Reopened PR #6

**Comment by rusfearuth** [comment]
@jeremistadler

It'll take me 5 hours to get to my laptop after I check PR.

Sorry for so long

вт, 19 сент. 2017 г. в 18:08, Jeremi Stadler <notifications@github.com>:

**Comment by rusfearuth** [comment]
PR had been merged to master. You can find fix at v1.2.1.

PS. Tnx for PR.
